packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -872,6 +872,11 @@ export interface CameraButtonStyles extends ControlBarButtonStyles {
 // @public
 export type ChatAdapter = ChatAdapterThreadManagement & AdapterState<ChatAdapterState> & Disposable & ChatAdapterSubscribers;
 
+// @public
+export type ChatAdapterOptions = {
```

Comment:
Should this be on the public stable API, or only on the Beta for now?

Reply:
Since the `createAzureCommunicationChatAdapterFromClient` is already public, this new type will also need to be public. 
Creating a new optional parameter after speaking with Nan should be ok.